### ISSUE TITLE
images use absolute paths instead of relative paths in search results

### ISSUE THREAD

**Issue by fournoas** [CloudCannon/pagefind#99]
I use relative paths to reference images in static web pages, just like:

```
──pages
    └───article
           01.jpg
           02.jpg
           index.html
```

`index.html`:

```
<img src="01.jpg">
```

but the image does not display correct in the search results.

Is there any possible to convert relative paths of image src to absolute paths in the search results?

**Comment by bglw** [comment]
👋 Hey @fournoas 

Not yet — #78 discusses a possible solution here but that has not been built out yet. Could you copy this over as a comment there so that I have everything in one place?

**Comment by bglw** [comment]
Moving ahead with @tabris17's suggestion in #78, will close this in favor of the existing issue 🙂

**Comment by bglw** [comment]
Hello! A processResult hook has been implemented in the [Pagefind v0.9.0](https://github.com/CloudCannon/pagefind/releases/tag/v0.9.0) release 🎉
